libs/user/types.ts

```diff
@@ -19,13 +19,22 @@ export interface UserOptions {
   zkpKeys: any; // TODO NightfallZkpKeys might have to be declared as class??;
 }
 
-export interface UserMakeDepositOptions {
+export interface UserMakeDeposit {
   tokenAddress: string;
   tokenStandard: string;
   value: string;
   feeGwei?: string;
 }
 
+export interface UserMakeWithdrawal extends UserMakeDeposit {
```

Comment:
Let's create one interface with a more generic name than UserMakeDeposit and extends from there for all others. What do you think?

Reply:
See commit [f2b6e3e](https://github.com/maticnetwork/nightfall-sdk/pull/66/commits/f2b6e3e85557707d9975eed3569e9bd4092f64cb)

Reply:
Awesome! Maybe we can create an interface MakeDeposits extends MakeTransaction {} and export this. But I dont see it a problem to merge! Let's go!

Reply:
But. Very nice!!